src/components/NewClusterModal.js

```diff
@@ -266,6 +282,10 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
       this.sendDeleteMetrics(shouldDeleteRuntime, this.willDeletePersistentDisk())
     }
 
+    if (shouldUpdateRuntime || shouldUpdatePersistentDisk) {
```

Comment:
This will catch PATCH calls, but not create/delete pairs, which we're also considering 'updates' from the user's perspective. We should consider moving this to `renderApplyButton` and handling it alongside the 'create' and 'delete' cases, to make sure the logic is consistent across them.

Reply:
Yeah I was thinking of 'create' and 'update' from the code perspective rather than from the users perspective, but if we just want to capture any update that comes from a user clicking 'update' (which does make sense) then I agree that the button is a better place to put this. My plan was to get this shipshape, merge it to mob-pd, and then build the 'create' case from the code I have in this PR as they're very similar.